sdks/python/setup.py

```diff
@@ -117,6 +117,7 @@ def get_version():
 
 REQUIRED_TEST_PACKAGES = [
     'nose>=1.3.7',
+    'numpy',
```

Comment:
numpy is already in `REQUIRED_PERF_TEST_PACKAGES`: https://github.com/apache/beam/blob/a5634128d194161aebc8d03229fdaa1066cf7739/sdks/python/setup.py#L132

Reply:
`REQUIRED_PERF_TEST_PACKAGES` are not included into `REQUIRED_TEST_PACKAGES`, however this PR comes with a unit test that exercises the coders microbenchmark, and this code will require `numpy`. That said, I think we still need `numpy` in `REQUIRED_TEST_PACKAGES`.